src/schema/v2/sales.ts

```diff
@@ -63,7 +63,8 @@ export const SalesConnectionField: GraphQLFieldConfig<void, ResolverContext> = {
       paginationArgs
     )
 
-    const loader = registered ? loaderWithoutCache : loaderWithCache
+    const loader =
+      typeof registered === "boolean" ? loaderWithoutCache : loaderWithCache
```

Comment:
This should be fine I think! I've used `Boolean` to coerce before:

<img width="175" alt="Screen Shot 2020-08-17 at 10 28 02 AM" src="https://user-images.githubusercontent.com/1457859/90407360-6f847000-e074-11ea-9f6b-25af5884b68d.png">

Reply:
In this case we actually want to distinguish "not specified"  from `true` or `false` as it means we don't filter at all based on the registration status. This brings up @erikdstock's concern about the API not being intuitive enough and we might want to switch to a more enum-based approach, such as:

 * `registered: ALL`: to return any sales or not filter at all
 * `registered: REGISTERED`: to return only the sales the user has registered for
 * `registered: UNREGISTERED`: to return only the sales the user has _not_ registered for

@erikdstock WDYT?